### ISSUE TITLE
PageReportMode() have no visible buttons Accept/Cancel under web

### ISSUE THREAD

**Issue by bambinoua** [jhomlala/catcher#129]
I use `PageReportMode` in my web project. When error occurred the page with error is displayed but there are no visible buttons `Accept` and `Cancel` to handle the page.
```dart
final debugOptions = CatcherOptions(PageReportMode(), [ConsoleHandler()]);
```

**Comment by jhomlala** [comment]
![image](https://user-images.githubusercontent.com/1329033/103363878-845e1f00-4abc-11eb-9088-880f0b1b2182.png)
The buttons are visible on my side. Maybe you've changed global theme of buttons? Anyway, feel free to copy PageReportMode and use it as custom report mode with custom buttons.

**Comment by drcdev** [comment]
I found this issue as well. The buttons are there, but the text is invisibile. In my case it is because I am using a darkMode theme. The DialogReportMode seems to inherit the theme as expected, but the PageReportMode does not. PageReportMode uses a default blue light theme, but takes the text brightness from the app level theme. It might be an easy fix to just apply the app theme to this scaffold and its widgets.

**Comment by drcdev** [comment]
Figured out how to pass the theme through. Will submit a PR :)

**Comment by jhomlala** [comment]
Fixed in 0.4.1.
